test/__init__.py

```diff
@@ -0,0 +1,6 @@
+import os
+import sys
+
+where_i_am = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(where_i_am + "/../pipelineutilities/pipelineutilities/")
+sys.path.append(where_i_am + "/../process_manifest/")
```

Comment:
These may need to be inserted back into initial calling tests and handler.py

Reply:
At this point, the tests are running both locally, on travis and i am anticipating code build since is closer to a command-line system.    But I don't understand why it is not called in all cases.